include/onnxruntime/core/framework/op_kernel.h

```diff
@@ -26,11 +26,15 @@
 #include "core/graph/graph_viewer.h"
 #include "core/graph/onnx_protobuf.h"
 #include "core/common/gsl.h"
+
 namespace onnxruntime {
 class OpKernelContext;
 }
 #endif
 
+#include "interface/framework/kernel.h"
```

Comment:
No need to include kernel.h here? I don't see any class declared in kernel.h in the changes of this file

Reply:
we probably shouldn't include interface/kernel here, as this file is for the internal kernel impl